packages/schematics/angular/migrations/update-7/typescript-helpers.ts

```diff
@@ -42,38 +38,27 @@ export function typeScriptHelpersRule(): Rule {
 }
 
 function _updateTsConfig(): Rule {
-  return (host: Tree) => {
+  return (host) => {
     const tsConfigPath = '/tsconfig.json';
-    const buffer = host.read(tsConfigPath);
-    if (!buffer) {
-      return host;
-    }
-
-    const tsCfgAst = parseJsonAst(buffer.toString(), JsonParseMode.Loose);
-    if (tsCfgAst.kind !== 'object') {
-      return host;
-    }
 
-    const compilerOptions = findPropertyInAstObject(tsCfgAst, 'compilerOptions');
-    if (!compilerOptions || compilerOptions.kind !== 'object') {
-      return host;
+    let tsConfigJson;
+    try {
+      tsConfigJson = new JSONFile(host, tsConfigPath);
+    } catch {
+      return;
     }
 
-    const importHelpers = findPropertyInAstObject(compilerOptions, 'importHelpers');
-    if (importHelpers && importHelpers.value === true) {
-      return host;
+    const compilerOptions = tsConfigJson.get(['compilerOptions']);
```

Comment:
This check here should be redundant with the new helpers.

Reply:
The original skipped the addition if `compilerOptions` was not present.  There is no test for that case so I was unsure of whether it was intentional but I left the behavior for consistency.